packages/base-styles/_mixins.scss

```diff
@@ -513,11 +510,23 @@
 	// Firefox 109+ and Chrome 111+
 	scrollbar-color: $handle-color $track-color; // Syntax, "dark", "light", or "#handle-color #track-color"
 	scrollbar-width: thin;
-	scrollbar-gutter: stable;
+	scrollbar-gutter: auto;
+
+	// Needed to fix a Safari rendering issue.
+	will-change: transform;
 
+	// On-hover/focus visibility.
 	&:hover,
 	&:focus,
 	& > * {
 		visibility: visible;
 	}
+
+	// Scrolling behavior.
+	overflow: hidden;
```

Comment:
I think this line could be causing a few issues with windowing and other logic in the list view.